tests/test_pkg_versions.py

```diff
@@ -0,0 +1,32 @@
+"""Test to check if style packages are in same versions as pre-commit config."""
+
+import configparser
+from pathlib import Path
```

Comment:
I was thinking why didn't you use this library anywhere else in this project instead of using os ?

Reply:
I copied this file from doctr ;-)